### DISCUSSION TITLE
Option to Suppress Source Code Archives in Release

### DISCUSSION THREAD

**Discussion by MrGadget1024** [semantic-release/semantic-release#3349] [Release workflow]
Currently we produce a Unity Package as part of the release. We'd like to suppress producing the other files if possible as an option in our [releaserc.yml](https://github.com/MirrorNetworking/Mirror/blob/master/.releaserc.yml).

![image](https://github.com/semantic-release/semantic-release/assets/9826063/71949c67-6c03-4d40-870f-c2c350cbd027)

**Comment by travi** [comment]
I believe those are added to releases automatically without an option to prevent them
